### ISSUE TITLE
Implement/connect error handling

### ISSUE THREAD

**Issue by gordonwoodhull** [att/rcloud.shiny#21]
Errors in user code either disappear or we get ghastly error messages that have nothing to do with user code, like

```
Error: send: Error in self$close() : attempt to apply non-function R trace: withReactiveDomain(
... 50 lines of mashed-together R code ...
```

Hypothesis: Shiny is doing error handling, but since we're not connected to it, the error gets swallowed. Sometimes Shiny crashed due to the disconnect, so the error message seen by the user is the Shiny crash.

**Comment by useless5771** [comment]
@gordonwoodhull I managed to provoke this error using the 'Observer demo' example notebook by adding a call to an undefined function. So I have something to begin with, but if you have some other examples that result in such errors they would definitely help me in investigating this issue.

Thanks

**Comment by gordonwoodhull** [comment]
Well! Thank **you** for documenting what context this happens in. 

I guess I was in a hurry and thought this bug was triggered for all errors in the code, but it seems to be only some reactive contexts, but not others.

My apologies for not doing more triage. I wrote rcloud.shiny but I think Shiny scares me.

**Comment by gordonwoodhull** [comment]
fixed by #22. thanks @useless5771!
